GWAS/LMM.ipynb

```diff
@@ -392,26 +392,30 @@
     "On a minimal working example (MWE) dataset,\n",
     "\n",
     "```\n",
-    "sos run LMM.ipynb regenie \\\n",
+    "sos run ~/bioworkflows/GWAS/LMM.ipynb regenie \\\n",
     "    --cwd output \\\n",
-    "    --bfile data/genotypes21_22.bed \\\n",
-    "    --maf-filter 0.001 \\\n",
-    "    --sampleFile data/imputed_genotypes.sample \\\n",
-    "    --bgenFile data/imputed_genotypes_chr*.bgen \\\n",
-    "    --phenoFile data/phenotypes.txt \\\n",
-    "    --formatFile data/regenie_template.yml \\\n",
-    "    --phenoCol ASTHMA T2D\\\n",
+    "    --bfile /SAY/dbgapstg/scratch/UKBiobank/MWE/genotypes21_22.bed \\\n",
```

Comment:
@dianacornejo please don't ever use absolute path for test command. Always use relative path. It is always a pain for me (and possibly for others) to change all the paths in order reproduce it on my local computer. I can live with  eg `MWE/imputed_genotypes.sample` but not `/SAY/dbgapstg/scratch/UKBiobank/MWE/imputed_genotypes.sample` -- please change the command to using relative paths.